frontend/src/components/VHeader/VHeaderInternal.vue

```diff
@@ -148,6 +151,13 @@ export default defineComponent({
       deactivateFocusTrap,
     })
 
+    const eventedOnTriggerClick = (...args) => {
```

Comment:
`onTriggerClick` does not take any arguments. I don't think we need `...args` here.